cluster-scope/base/core/namespaces/esi-operate-first-project/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: esi-operate-first-project
+    annotations:
+        openshift.io/description: Our project permits multiple tenants to flexibly allocate baremetal machines from a pool of available hardware, create networks, attach baremetal nodes and networks, and to optionally provision an operating system on those systems through the use of an associated provisioning service.
```

Comment:
Actually, can we add a `display-name` here?

Reply:
renamed to `display-name`